cookbook/specs/spec_add_side.json

```diff
@@ -0,0 +1,489 @@
+                        "stateful": 0
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "side.gmm.QueryService/Pool",
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": true,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 0
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "side.gmm.QueryService/Volume24",
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": true,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 0
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "side.gmm.QueryService/TotalVolume",
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": true,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 0
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "side.gmm.QueryService/APR",
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": true,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 0
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "side.interchainquery.v1.QueryService/PendingQueries",
```

Comment:
using grpcurl and KJnodes endpoint, I didn't see this API supported. where did you get it from?
If it's old and deprecated, remove it here and in REST

Reply:
I got it from there GitHub proto file so which one should I follow the ones from gcurl or the proto files?

Reply:
i tried 3 different endpoints and none support this. I think you should remove it for now

Reply:
This changed with the S2-testnet-1